pkg/util/math.go

```diff
@@ -1,33 +1,25 @@
 package util
 
+import (
```

Comment:
There are only handful references for each of these functions, I'd suggest replacing them with new functions and deleting this file.